From 8fa78e87f1f4bd2bb27ecf3920c83aa59cee6079 Mon Sep 17 00:00:00 2001
From: Jeroen Hofstee <jhofstee@victronenergy.com>
Date: Mon, 22 May 2017 17:36:59 +0200
Subject: [PATCH] ccgx: make the fbcon a module

The console will also open the input devices and hence interfere
with the Qt application doing the same, so make fbcon a module which
can be probed if the intent is to use the graphical console.

Note: this blanks the screen during boot instead of showing a logo,
so a better solution must be found for this...
---
 arch/arm/configs/omap2plus_defconfig | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/configs/omap2plus_defconfig b/arch/arm/configs/omap2plus_defconfig
index a6a9ad7..2622f54 100644
--- a/arch/arm/configs/omap2plus_defconfig
+++ b/arch/arm/configs/omap2plus_defconfig
@@ -2744,7 +2744,7 @@ CONFIG_HDMI=y
 # Console display driver support
 #
 CONFIG_DUMMY_CONSOLE=y
-CONFIG_FRAMEBUFFER_CONSOLE=y
+CONFIG_FRAMEBUFFER_CONSOLE=m
 CONFIG_FRAMEBUFFER_CONSOLE_DETECT_PRIMARY=y
 CONFIG_FRAMEBUFFER_CONSOLE_ROTATION=y
 CONFIG_LOGO=y
@@ -4537,7 +4537,7 @@ CONFIG_DDR=y
 # CONFIG_IRQ_POLL is not set
 CONFIG_LIBFDT=y
 CONFIG_OID_REGISTRY=y
-CONFIG_FONT_SUPPORT=y
+CONFIG_FONT_SUPPORT=m
 CONFIG_FONTS=y
 CONFIG_FONT_8x8=y
 CONFIG_FONT_8x16=y
-- 
2.7.4

